src/Test/config.jl

```diff
@@ -12,7 +12,7 @@ struct TestConfig{T <: Number}
     optimal_status::MOI.TerminationStatusCode
     basis::Bool # can get variable and constraint basis status
     function TestConfig{T}(;
-        atol::Float64 = 1e-8, rtol::Float64 = 1e-8, solve::Bool = true,
+        atol::Real = 1e-8, rtol::Real = 1e-8, solve::Bool = true,
```

Comment:
I think `atol = Base.rtoldefault(one(T), one(T), 0)` and `rtol = atol` would make `atol` and `rtol` have type `T`.

Reply:
`sqrt(eps(T))` gives the same result and type, and is shorter and more transparent, so I used that.